src/Microsoft.CodeAnalysis.Razor/ViewComponentTypes.cs

```diff
@@ -21,6 +21,8 @@ internal static class ViewComponentTypes
 
         public const string Task = "System.Threading.Tasks.Task";
 
+        public const string IDictionary = "System.Collections.Generic.IDictionary`2";
```

Comment:
I originally moved this functionality to the MVC assembly, and I had to do this when I did that. Personally this is one of those 'just do something sensible' cases. The full name of `IDictionary` is not going to change.